ProcessLib/HydroMechanics/HydroMechanicsFEM-impl.h

```diff
@@ -390,16 +426,35 @@ void HydroMechanicsLocalAssembler<ShapeFunctionDisplacement,
         auto const& N_p = _ip_data[ip].N_p;
         auto const& dNdx_p = _ip_data[ip].dNdx_p;
 
-        double const S = _process_data.specific_storage(t, x_position)[0];
         double const K_over_mu =
             _process_data.intrinsic_permeability(t, x_position)[0] /
             _process_data.fluid_viscosity(t, x_position)[0];
         auto const alpha_b = _process_data.biot_coefficient(t, x_position)[0];
-        auto const rho_fr = _process_data.fluid_density(t, x_position)[0];
+        // TODO get fluid properties from GPML
+        double const rho_fr =
```

Comment:
Looks like the same initialization as above? Maybe you could re-use a small function?

Reply:
> @nagelt Regarding the E, nu; Could be implemented in the MechanicsBase interface.

There is a general way to obtain K_S (drained bulk modulus) that should work for all material models including anisotropic and inelastic ones (in case of anisotropy this will be an upper bound):

In Kelvin-representation we take 1/9 * I^T C I, with I being the second-order identity, and C our consistent material tangent. In 3D tensor notation this would be 1/9 \delta_{ij} C_{ijkl} \delta_{kl}.

Could maybe be a default-implementation unless someone specifies an alternative.